Source/FieldSolver/ImplicitSolvers/ImplicitSolver.H

```diff
@@ -83,12 +83,9 @@ public:
     virtual void ComputeRHS ( WarpXSolverVec&  a_RHS,
```

Comment:
I notice that the comments do not have the list of input parameters for the routines. This is not for this PR, but can be added in the future.